cmd/juju/status/status_test.go

```diff
@@ -1814,7 +1826,7 @@ var statusTests = []testCase{
 
 		// scoped on wordpress/0
 		scopedExpect{
-			"subordinates scoped on logging",
```

Comment:
I think this was correct? The subordinate is scoped to the logging relation of the wordpress application?